modules/ROOT/examples/live-demos/context-menu/index.js

```diff
@@ -2,8 +2,8 @@ tinymce.init({
   selector: 'textarea#context-menu',
   height: 500,
   plugins: [
-    'link image imagetools table spellchecker lists'
```

Comment:
I wasn't sure whether to change this to `tinymcespellchecker` or to just remove it. I have just removed it for now but am happy to change it back

Reply:
Given this is only meant to be a basic demo, I think removing it is the right option. We really could go either way though 🤷